wandb/sdk/internal/progress.py

```diff
@@ -49,6 +49,13 @@ def rewind(self):
         self.bytes_read = 0
         self.file.seek(0)
 
+    def __getattr__(self, name):
+        """Fallback to the file object for attrs not defined here"""
+        if hasattr(self.file, name):
+            return getattr(self.file, name)
+        else:
+            raise AttributeError
```

Comment:
Add an error message?

Reply:
This should raise the default `AttributeError` message, i.e. `has no attribute 'foo'`